Integrations/MicrosoftGraphListener/MicrosoftGraphLIstener.py

```diff
@@ -0,0 +1,982 @@
+            },
+            'comment': comment
+        }
+
+    def _get_attachment_mime(self, attachment_id):
+        """
+        Gets attachment mime.
+
+        :type attachment_id: ``str``
+        :param attachment_id: Attachment id to get MIME
+
+        :return: The MIME of the attachment
+        :rtype: ``str``
+        """
+        suffix_endpoint = f'users/{self._mailbox_to_fetch}/messages/{attachment_id}/$value'
+        mime_content = self._http_request('GET', suffix_endpoint, resp_type='text')
+
+        return mime_content
+
+    def _get_email_attachments(self, message_id):
+        """
+        Get email attachments  and upload to War Room.
+
+        :type message_id: ``str``
+        :param message_id: The email id to get attachments
+
+        :return: List of uploaded to War Room data, uploaded file path and name
+        :rtype: ``list``
+        """
+
+        attachment_results = []  # type: ignore
+        suffix_endpoint = f'users/{self._mailbox_to_fetch}/messages/{message_id}/attachments'
+        attachments = self._http_request('Get', suffix_endpoint).get('value', [])
+
+        for attachment in attachments:
+            attachment_type = attachment.get('@odata.type', '')
+            attachment_name = attachment.get('name', 'untitled_attachment')
+            if attachment_type == self.FILE_ATTACHMENT:
+                try:
+                    attachment_content = base64.b64decode(attachment.get('contentBytes', ''))
+                except Exception as e:  # skip the uploading file step
+                    demisto.info(f"MS-Graph-Listener: failed in decoding base64 file attachment with error {str(e)}")
+                    continue
+            elif attachment_type == self.ITEM_ATTACHMENT:
+                attachment_id = attachment.get('id', '')
+                attachment_content = self._get_attachment_mime(attachment_id)
+                attachment_name = f'{attachment_name}.eml'
+            # upload the item/file attachent to War Room
+            upload_file(attachment_name, attachment_content, attachment_results)
+
+        return attachment_results
+
+    def _parse_email_as_incident(self, email):
+        """
+        Parses fetched emails as incidents.
+
+        :type email: ``dict``
+        :param email: Fetched email to parse
+
+        :return: Parsed email
+        :rtype: ``dict``
+        """
+        parsed_email = MsGraphClient._parse_item_as_dict(email)
+
+        if email.get('hasAttachments', False):  # handling attachments of fetched email
+            parsed_email['Attachments'] = self._get_email_attachments(message_id=email.get('id', ''))
+
+        incident = {
+            'name': parsed_email['Subject'],
+            'details': email.get('bodyPreview', '') or parsed_email['Body'],
+            'labels': MsGraphClient._parse_email_as_labels(parsed_email),
+            'occurred': parsed_email['ReceivedTime'],
```

Comment:
should this be `ModifiedTime`?

Reply:
Changed the logic to fetch by modified time, so manual email moving into fetch folder will be supported also...